### DISCUSSION TITLE
希望增加导出图片，并切割成多个图片，方便发小红书、x等平台

### DISCUSSION THREAD

**Discussion by wu0407** [doocs/md#473] [Ideas]
图片比例4:3 1:1等

**Comment by zillionare** [comment]
你可以试试 MarkTwain (https://md.jieyu.ai/)， 支持分页预览，页面比例可自定义，还可以通过模板和自定义样式，来设置封面页。在分页预览模型下，导出为 png，将自动导出为多页图片。
